selfdrive/test/test_updated.py

```diff
@@ -53,7 +54,7 @@ def setUp(self):
       f"cd {self.basedir} && scons -j{os.cpu_count()} cereal/ common/"
     ])
 
-    self.params = Params(db=os.path.join(self.basedir, "persist/params"))
```

Comment:
I'm pretty sure this was a bug in the test, since the parameter seems to be named `d`